snappy_pipeline/workflows/ngs_mapping/__init__.py

```diff
@@ -362,28 +362,29 @@
     # Configuration for STAR
     star:
       path_index: REQUIRED # Required if listed in ngs_mapping.tools.rna; otherwise, can be removed.
+      path_features: ""    # Required for computing gene counts
```

Comment:
How is it related to `quant_mode`?

Reply:
The configuration files doesn't know about `quant_mode`. If `path_features` is defined, the pipeline will compute gene counts based on the features defined in the `gtf` file. When it's missing, no gene counts are produced.

The `quant_mode` variable in the wrapper code is just a trick to tell `STAR` what additional output it must produce (gene counts and/or mappings on the transcriptome)